src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessorConstants.java

```diff
@@ -215,6 +215,14 @@ public interface UserPrefsAccessorConstants extends Constants {
    @DefaultStringValue("The number of spaces to insert when pressing the Tab key.")
    String numSpacesForTabDescription();
 
+   /**
+    * The editor scroll sensitivity.
+    */
+   @DefaultStringValue("Editor scroll sensitivity")
+   String editorScrollMultiplierTitle();
+   @DefaultStringValue("The editor scroll speed")
+   String editorScrollMultiplierDescription();
```

Comment:
Again, possibly could use a more descriptive description?

Reply:
Even just Editor scroll speed or Editor scroll speed sensitivity might be clearer for the title

Reply:
I kind of followed how VS Code names their preference. Adding speed into the description is a good idea though. Speed and sensitivity are probably the two common words for scrolling.